### PR TITLE
feat: Implement `stream` interface

### PR DESCRIPTION
**Pull request by apepkuss**
Major changes:

- Rename `wasm_chat.py` to `llama_edge.py`
- Rename the `WasmChatService` class to `ChatService`
- Implement the `stream` interface for `ChatService`
- Add `test_chat_wasm_service_streaming` in the integration test
- Update `llama_edge.ipynb`

### CODE REVIEW

File: libs/community/langchain_community/chat_models/llama_edge.py [hunk 1]
```diff
@@ -45,10 +53,26 @@ def _convert_message_to_dict(message: BaseMessage) -> dict:
...
+        return HumanMessageChunk(content=content)
+    elif role == "assistant" or default_class == AIMessageChunk:
+        return AIMessageChunk(content=content)
+    elif role or default_class == ChatMessageChunk:
+        return ChatMessageChunk(content=content, role=role)
+    else:
+        return default_class(content=content)
+
+
+class ChatService(BaseChatModel):
```
Comment (hwchase17) [inline libs/community/langchain_community/chat_models/llama_edge.py:72]: should this be named LlamaEdgeChatService to make it more specific? ChatService is kinda generic

Comment (apepkuss) [reply libs/community/langchain_community/chat_models/llama_edge.py:72]: Sure! `LlamaEdgeChatService` sounds better. Will update the name. Thanks a lot!

**Comment by apepkuss** [comment]
@hwchase17 Please help review the latest changes. Thanks a lot!

**Review by Undertone0809 — APPROVED** [review]

**Comment by apepkuss** [comment]
@hwchase17 Thanks for the review!

**Comment by apepkuss** [comment]
@Undertone0809 Thanks for the review!

**Review by hwchase17 — APPROVED** [review]
thanks!

**Pull request merged**